lecture06.tex

```diff
@@ -58,7 +69,8 @@ \section{Lecture 6}
   By the above, we have $\afrak \neq \afrak \pfrak^{-1}$, and $\pfrak \neq \pfrak \pfrak^{-1} \subset \Ocal$.
   Since $\pfrak$ is maximal, we find $\pfrak \pfrak^{-1} = \Ocal$.
 
-  The maximality of $\afrak$ shows that the \todo{ideal}{Why is this an ideal?} $\afrak \pfrak^{-1}$ admits a factorization as $\pfrak_{1} \cdots \pfrak_{k}$.
+  The maximality of $\afrak$ shows that the ideal $\afrak \pfrak^{-1}$ admits a factorization as $\pfrak_{1} \cdots \pfrak_{k}$.
+  We know $\afrak \pfrak^{-1}$ is an ideal as $\afrak \subseteq \pfrak$, and so $\afrak \pfrak^{-1}\subseteq \pfrak \pfrak^{-1} \subseteq \Ocal.$
```

Comment:
```suggestion
  We know $\afrak \pfrak^{-1}$ is an ideal as $\afrak \subset \pfrak$, and so $\afrak \pfrak^{-1}\subset \pfrak \pfrak^{-1} \subset \Ocal.$
```
I prefer $\subset$ for possibly-non-strict inclusion and $\subsetneq$ for strict inclusion.